youtube_dl/YoutubeDL.py

```diff
@@ -161,6 +162,7 @@ class YoutubeDL(object):
     playlist_items:    Specific indices of playlist to download.
     playlistreverse:   Download playlist items in reverse order.
     playlistrandom:    Download playlist items in random order.
+    playlistprompt:    Ask for each playlist file to download
```

Comment:
@dstftw You're right, using a prompt via a programmatic interface won't be possible. What do I need to do to make it clear that this parameter should only be used on the command line in an interactive session?